cluster/gce/gci/configure-helper-common.sh

```diff
@@ -1750,7 +1757,7 @@ function compute-master-manifest-variables {
   fi
 
   INSECURE_PORT_MAPPING=""
-  if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "true" ]]; then
+  if [[ "${USE_INSECURE_SCALEOUT_CLUSTER_MODE:-false}" == "true" ]]; then
```

Comment:
good catch. will change. this might not impact our testing since the KUBE_ENABLE_APISERVER_INSECURE_PORT
 is not set to true always to get the metrics collection working. however once we fixed the metric collection port to be secured. it will cause test failures. thanks for catching this up.

Reply:
fixed.